### PR TITLE
Updates from the world of Seam

### PR DESCRIPTION
**Pull request by ghost**
processes/development/new-app-setup/README.md
   processes/development/new-app-setup/semaphore-production-deploy-config
   processes/development/new-app-setup/semaphore-staging-deploy-config

### CODE REVIEW

File: processes/development/new-app-setup/semaphore-staging-deploy-config [hunk 1]
```diff
@@ -1,31 +1,44 @@
...
 sudo dpkg -i ./mongodb-org-shell_3.2.6_amd64.deb
-
 mongodump --host $PRODUCTION_DB_HOST --port $PRODUCTION_DB_PORT --db $PRODUCTION_DB_NAME -u $PRODUCTION_DB_USERNAME -p $PRODUCTION_DB_PASSWORD -o mongodump
 echo 'db.getCollectionNames().forEach(function(c) {if (c.indexOf("system.") == -1) {db.getCollection(c).drop();}})' | mongo $STAGING_DB_HOST:$STAGING_DB_PORT/$STAGING_DB_NAME -u $STAGING_DB_USERNAME -p $STAGING_DB_PASSWORD
 mongorestore -h $STAGING_DB_HOST:$STAGING_DB_PORT -d $STAGING_DB_NAME -u $STAGING_DB_USERNAME -p $STAGING_DB_PASSWORD mongodump/*
-
 # Copy production S3 data to staging
-
-export AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID
-export AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY
```
Comment (pauldowman) [inline processes/development/new-app-setup/semaphore-staging-deploy-config:25]: That was looking a bit redundant. 😀

File: processes/development/new-app-setup/semaphore-staging-deploy-config [hunk 2]
```diff
@@ -1,31 +1,44 @@
...
+exp login -u $EXPO_STAGING_USERNAME -p $EXPO_STAGING_PASSWORD
+exp whoami
+cd mobile
+mv app-staging.json app.json
+cat app.json
+npm install --global sentry-cli-binary
+npm install
+npm --version
+node --version
+exp --version
```
Comment (pauldowman) [inline processes/development/new-app-setup/semaphore-staging-deploy-config:42]: Would we want these three `xx --version` lines for the production deploy too? I realize they aren't necessary but they're helpful for debugging.

Comment (pauldowman) [inline processes/development/new-app-setup/semaphore-staging-deploy-config:42]: Would we want these three xx --version lines for the production deploy too? I realize they aren't necessary but they're helpful for debugging.

**Review by pauldowman — APPROVED** [review]
👍

File: processes/development/new-app-setup/semaphore-staging-deploy-config (hunk 2 above)
Comment (ghost) [reply processes/development/new-app-setup/semaphore-staging-deploy-config:42]: Yep. Done.

File: processes/development/new-app-setup/semaphore-build-settings [hunk 3]
```diff
@@ -0,0 +1,17 @@
+cd web/
+curl https://install.meteor.com/ | sh
+meteor --version
+meteor npm install
+cp /home/runner/settings.json settings.json
+meteor npm run lint
+meteor npm run unit-test
+npm install -g exp
```
Comment (cfnelson) [inline processes/development/new-app-setup/semaphore-build-settings:8]: @ccuilla Shouldn't we split the script out for web and mobile builds? Or did I mistakenly miss where the guide says this is only for mobile builds?

**Review by cfnelson — COMMENTED** [review]
@ccuilla Sorry, I thought I left a review but I ended up leaving a comment. I wanted to know if the semaphore build script should mention that it's only for mobile? As we don't keep a `web` folder if only creating a web app.

Hopefully when you are back we can finish/polish this off and get it merged. It looks very detailed. 👍

File: processes/development/new-app-setup/README.md [hunk 4]
```diff
@@ -275,17 +265,13 @@ Atlas is the best option if wishing to scale CPU & Memory independently to it's
     1. NOTE: Paul needs to do this step currently. Will fix...
 1. Edit Build Settings:
     1. Go to [https://semaphoreci.com/okgrow/](https://semaphoreci.com/okgrow/)&lt;PROJECT_NAME&gt;/settings
-    1. Set the Node version to node.js 4.8.0
-    1. Add each of these lines under setup
-        1. `curl https://install.meteor.com/ | sh`
-        1. `meteor --version`
-        1. `meteor npm install`
-        1. `meteor npm start` _wip: find and add instructions to listen for success and exit process_
+    1. Set the Node version to node.js 7.10.1 (or later)
```
Comment (karldanninger) [inline processes/development/new-app-setup/README.md:268]: As of Meteor 1.6, can we change this to 8.7.x now?

**Review by karldanninger — APPROVED** [review]
Would love to get this merged in today 👍

File: processes/development/new-app-setup/README.md (hunk 4 above)
Comment (ghost) [reply processes/development/new-app-setup/README.md:268]: @karldanninger Yes.

**Pull request merged**